plugins/org.eclipse.osee.framework.core/src/org/eclipse/osee/framework/core/data/OrcsTokenServiceImpl.java

```diff
@@ -46,13 +55,25 @@ public final class OrcsTokenServiceImpl implements OrcsTokenService {
    private final Map<Long, Tuple3Type<?, ?, ?>> tuple3Types = new ConcurrentHashMap<>();
    private final Map<Long, Tuple4Type<?, ?, ?, ?>> tuple4Types = new ConcurrentHashMap<>();
 
+   private final List<String> convertedRelations = new ArrayList<>();
    /**
     * Register core types first to prevent their ids from being registered by mistaken or malicious code
     */
    public OrcsTokenServiceImpl() {
       artifactTypes.put(ArtifactTypeToken.SENTINEL.getId(), ArtifactTypeToken.SENTINEL);
       attributeTypes.put(AttributeTypeToken.SENTINEL.getId(), AttributeTypeToken.SENTINEL);
       relationTypes.put(RelationTypeToken.SENTINEL.getId(), RelationTypeToken.SENTINEL);
+      File relFile = OseeInf.getResourceAsFile("relations/relations.json", OrcsTypeTokens.class);
```

Comment:
I thought we said we wanted one per - relation type java file...are we going to do that later?

Reply:
I don't remember that... however, at the very least we will need one for the open source relations, lba relations and f18 relations...

Reply:
scratch that we only need 1, cause we are using Ids